assets/app/views/_templateopt.html

```diff
@@ -0,0 +1,21 @@
+<div class="template-options gutter-bottom" ng-show="hasParameters">
+  <div class="flow">
+    <div class="flow-block">
+      <h2>Template options</h2>
```

Comment:
Do we want to call this "Parameters" for closer connection to the underlying concept? "Parameters" doesn't seem too unapproachable